Core/Src/main.c

```diff
@@ -87,17 +98,47 @@ int main(void)
   /* Initialize all configured peripherals */
   MX_GPIO_Init();
   MX_USART2_UART_Init();
+  MX_SPI1_Init();
+  MX_TIM1_Init();
   /* USER CODE BEGIN 2 */
 
+  // SPI
+  SPI_HandleTypeDef *pSPI_Handle = &hspi1;
+  uint8_t TxBuffer[3] = {0x01, 0x80, 0x00};
+  uint8_t RxBuffer[3] = {0, 0, 0};
```

Comment:
so Tx will never change so it will be const... make sure to also make the name ALL CAPS to follow const convention